.github/workflows/deflake.yml

```diff
@@ -0,0 +1,28 @@
+name: Deflake
+on:
+  schedule:
+    - cron: '22 18 * * 0'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      K8S_VERSION: "1.21.x"
```

Comment:
you can remove this env block and just take the default `K8S_VERSION` from the Makefile